test/demo-logger-using-pvc/README.md

```diff
@@ -0,0 +1,52 @@
+# Containerized-Distributed-Logger 📝
+Logging Service to demonstrate the distributed logging with log collector.
+***
+
+### ⚽ Current Functionality:
+  - logger.py periodically logs the demo statements at /var/log/demoLogger
+    in the container which automatically gets copied to the /var/log/demoLogger
+    file on the node using PV and PVC. ([EOS-24413](https://jts.seagate.com/browse/EOS-24413))
+
+### 🥅 Next Things to Acheive:
+  - Deploy glusterfs to have a replicated/distributed log file system. ([EOS-24410](https://jts.seagate.com/browse/EOS-24410))
```

Comment:
CORTX services doesnt need to bother about log replication or even shared distributed shared storage, that will be made available by admin as part of Solution architecture, as CORTX pre-requisite

Reply:
sure. removed the mention.